src/sass/phone-input/phone-input.scss

```diff
@@ -10,7 +10,14 @@
     border-radius: 8px 0 0 8px;
     min-width: 64px;
     padding-inline: var(--spacing-200) var(--spacing-100);
-    border-right: none;
+    border-right-style: revert;
```

Comment:
This looks to be causing a display issue where the line is much darker than it used to be. 

<kbd><img width="374" alt="image" src="https://github.com/eBay/skin/assets/1675667/b4c81bc8-b6e2-4c3e-8d3b-c43dc8d49068"></kbd>

Reply:
My fault for only checking in dark mode 🙈. Switched to `none`.